chart/values.schema.json

```diff
@@ -2856,6 +2856,11 @@
                     "type": "array",
                     "default": []
                 },
+                "uid": {
```

Comment:
I believe this should come as part of .Values.redis.uid parameter ? 

For compatibiity with previous version also I tihnk this should have `None` default and the whole `security-context` section should be conditional based on the uid set, I think because someone upgrading the chart will already have a root-owned db in /data abwhen persistence is enabled and it might get non-accessible if "runAsUser" is used.

Reply:
Of course `.Values.redis.uid` , that was an oversight. 

I see the issue with upgrading with existing redis persistence. What do you think of solution below?

Reply:
Looks good. One more thing - we also have unit tests for the chart (chart/tests) - they are rather simple - you pass parameters and check if the rendered templates are as expected. It would be great to add tests for this change there.

Reply:
And I also recommend installing pre-commit - I think it will then generate documentation automatically from the schema (see https://github.com/apache/airflow/blob/main/STATIC_CODE_CHECKS.rst#pre-commit-hooks)